src/Controllers/VoyagerBreadController.php

```diff
@@ -246,6 +246,17 @@ public function getContentBasedOnType($request, $slug, $row){
       /********** FILE TYPE **********/
     } else if($row->type == 'file'){
 
+        $file = $request->file($row->field);
+        $filename = str_random(20);
+
+        $path =  $slug . '/' . date('F') . date('Y') . '/';
+
+        $full_path = $path . $filename . '.' . $file->getClientOriginalExtension();
+
+        Storage::put(config('voyager.storage.subfolder') . $path . $filename . '.' . $file->getClientOriginalExtension(), (string)$file, 'public');
```

Comment:
Why not just use `$full_path` here instead of duplicating `$path . $filename . '.' . $file->getClientOriginalExtension()`?